proto/p4/v1/p4runtime.proto

```diff
@@ -471,6 +471,8 @@ message MulticastGroupEntry {
   // Arbitrary metadata from the controller that is opaque to the target.
   // Added in 1.4.0.
   bytes metadata = 3;
+  // Backup replicas used as a fallback port.
+  repeated Replica backups = 4;
```

Comment:
I think nowadays we add a comment for new fields, to indicate which version they were introduced in. There should be examples in the file.

Reply:
I set the version to 1.4.2. I was not too sure where that comment would go for adding a new proto message.